cves/CVE-2013-2071.yml

```diff
@@ -61,13 +68,13 @@ fixes_vcc_instructions: |
 fixes:
    - commit: f505d993d47d75d762c632cef6a622928ed4bcd6
      note: SVN rev 1471372, from the Tomcat website.
-   - commit:
-     note:
+   - commit: 4171bafd2b38be19a6358ba75b9d71b6dbf3072b
+     note: SVN rev 1471371, from the Tomcat website.
 vccs:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: 855137c63bce55c7e70d21c30fddf2a9d165d067
+    note: Added too narrow error handling around the vulnerability.
+  - commit: 587b1a807129c2ab41623358281aa19ff2ff9950
+    note: Commit which originally introduced some of the exception handling.
```

Comment:
Did both of these commits have the too narrow handling? I could just be misunderstanding the notes for these commits, but it sounds like only the first-listed commit (855137c63bce55c7e70d21c30fddf2a9d165d067) is the one that had the too narrow handling problem. So is the second-listed one (587b1a807129c2ab41623358281aa19ff2ff9950) actually a VCC?

Reply:
So within the AsyncContextImpl (where the vulnerability existed) there were multiple calls that could potentially throw a runtime exception. Commit (587b1a8) was a refactoring that originally introduced some of the code that needed exception handling (it also included some narrow handling). Then the (855137c) commit added handling around more calls that needed it, but it was still too narrow (only catching IOException).

I reordered these commits and slightly adjusted the wording to hopefully make this clearer.